lib/generators/atomic_cms/scaffold/scaffold_generator.rb

```diff
@@ -0,0 +1,68 @@
+require 'active_support/core_ext/string'
+
+module AtomicCms
+  module Generators
+    class ScaffoldGenerator < Rails::Generators::Base
+      argument :model_name, required: true
+
+      desc <<-DESC.strip_heredoc
+        Create a model, controller, view, admin resource, and route.
+      DESC
+
+      source_root File.expand_path("../../templates", __FILE__)
+
+      def install_model_template
+        setup_scaffold
+        create_template({
+          template_name: 'model.erb',
+          full_path: "app/models/#{@scaffold.model_file_name}.rb"
+        })
+      end
+
+      def install_controller_template
+        create_template({
+          template_name: 'controller.erb',
+          full_path: "app/controllers/#{@scaffold.controller_file_name}_controller.rb"
+        })
+      end
+
+      def install_admin_template
+        create_template({
+          template_name: 'admin.erb',
+          full_path: "app/admin/#{@scaffold.model_file_name}.rb"
+        })
+      end
+
+      def install_show_view_template
+        create_template({
+          template_name: 'show.html.slim',
```

Comment:
Use 2 spaces for indentation in a hash, relative to the first position after the preceding left parenthesis.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.